cmd/init.go

```diff
@@ -20,19 +20,34 @@ import (
 	"github.com/spf13/cobra"
 )
 
+var (
```

Comment:
MUST: Please check up on this.  Can you put these directly into an AppConfig Struct, and then pass that AppConfig struct directly to the marshal function in root.go

Reply:
Chatted with Ryan.  He's going to go back and refactor the init anyways.